src/main/java/io/seqera/tower/cli/responses/RunDeleted.java

```diff
@@ -0,0 +1,17 @@
+package io.seqera.tower.cli.responses;
+
+public class RunDeleted extends Response {
```

Comment:
The command `towr --json runs delete ...` fails on binary CLI.

Reply:
Can't replicate this error locally using the binary build.